### ISSUE TITLE
Update to newsfeed and welcome email

### ISSUE THREAD

**Issue by kerrywatson1** [Anaconda-Server/docs.anaconda.org#188]
Matt Thornell said: 
> I did register for an Anaconda Cloud account though, and the newsfeed/ homepage says "Welcome to anaconda.org". Also the welcome email has the same language. I hope this will be part of the changes in place before the webinar [on 11/12].

Please change newsfeed and welcome email 
FROM "Welcome to anaconda.org"
TO "Welcome to Anaconda Cloud"

per Michele Chambers.

@stephenakearns

**Comment by srossross** [comment]
@kerrywatson1 this is unassigned and in the validate stage. Does this belong here?

**Comment by kerrywatson1** [comment]
Yow, yes, it needs to be done. I've assigned it to @mistakevin.

**Comment by srossross** [comment]
@kerrywatson1 It sounds like this issue still needs to be started. In that case it should go into the "New Issues" pipeline

Here is our documentation on how we are using the zenhub pipelines: https://docs.google.com/document/d/1-yl3MSjwrnAFg9oAy-7xuu28mo085xLojI61aaQSgCg/edit

**Comment by kerrywatson1** [comment]
Sean, I have no idea why it's in the validate pipeline. Nice doc btw, we
are enthusiastic users of Zenhub.

If you need to add this to the next sprint, please do. If you need me to
drag it from the validate column I can do that, too. Not sure what you are
looking for from me, I was just the messenger. Michele requested this
@stephenakearns

Kerry


_______________________________

Kerry Watson
Documentation and web project manager

[image: Continuum Analytics]
<http://t.sidekickopen29.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJN7t5XZsRzsQHMRbPXMfDgQnW2BFNn256dBBnf974Gsn02?t=http%3A%2F%2Fcontinuum.io%2F&si=5890486390751232&pi=c16f96ac-82e0-4cd5-a02d-ec4f8fd40549>

**Comment by stephenakearns** [comment]
@dsludwig - I think you worked on this with me while in Austin - can you attach the PR associated with this. @kerrywatson1 , monday, this fix will get released into prod - could you have someone on your team retest at that point to confirm the fix?

**Comment by kerrywatson1** [comment]
@electronwill will test it on Monday. Thanks team!

Kerry


_______________________________

Kerry Watson
Documentation and web project manager

[image: Continuum Analytics]
<http://t.sidekickopen29.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJN7t5XZsRzsQHMRbPXMfDgQnW2BFNn256dBBnf974Gsn02?t=http%3A%2F%2Fcontinuum.io%2F&si=5890486390751232&pi=2d54231e-bd77-4fd2-f01a-44d8eb255b42>

**Comment by dsludwig** [comment]
The email sender name and contents were fixed in: Anaconda-Server/anaconda-server#1049

**Comment by electronwill** [comment]
Currently it still says anaconda.org.

Delivered-To: wwarner+4@continuum.io
From: "Anaconda.org" <noreply@anaconda.org>
To: wwarner+4@continuum.io
Subject: Welcome, wwarner4 to Anaconda.org
Date: Mon, 7 Dec 2015 23:46:53 +0000

Thanks for signing up for anaconda.org!

Please click the link below to confirm activation of your anaconda.org account:

https://anaconda.org/account/confirm_email?code=x37efb806fba0ffcf&email=wwarner%2B4%40continuum.io

anaconda.org is currently an early-release service.  We appreciate your feedback, questions and suggestions.  As appropriate, we encourage you to use the anaconda.org issue tracker or email our support team.

You may find the following resources helpful as you familiarize yourself with anaconda.org:
Documentation: http://docs.anaconda.org
Issue Tracker: https://github.com/Anaconda-Server/docs.anaconda.org/issues
Email Support: support@anaconda.org

Sincerely, The anaconda.org team

**Comment by stephenakearns** [comment]
Hi @dsludwig is there any chance that 1049 above hasn't been deployed to prod yet?

**Comment by dsludwig** [comment]
Correct, this change will be released with 2.11 (currently in Beta)

**Comment by dsludwig** [comment]
This is released now, I have verified by creating a new user.

```
From: Anaconda Cloud <noreply@anaconda.org>
To: dludwig@continuum.io
Subject: Welcome, dludwig-test to Anaconda Cloud
Date: Thu, 10 Dec 2015 21:24:29 +0000

Thanks for signing up for Anaconda Cloud!

Please click the link below to confirm activation of your Anaconda Cloud account:

https://anaconda.org/account/confirm_email?code=3bfaf870ba2d8c6c&email=dludwig%40continuum.io

Anaconda Cloud is currently an early-release service.  We appreciate your
feedback, questions and suggestions.  As appropriate, we encourage you to use
the Anaconda Cloud issue tracker or email our support team.

You may find the following resources helpful as you familiarize yourself with
Anaconda Cloud:
Documentation: http://docs.anaconda.org
Issue Tracker: https://github.com/Anaconda-Server/support/issues
Email Support: support@anaconda.org

Sincerely, The Anaconda Cloud team
```
